Aptoide/aptoide-client-v8
# Feature/mob 512 apps name ab test

**What does this PR do?**

This PR implements a new AB test for the name of the apps bottom navigation item;

**Database changed?**
No

**Where should the reviewer start?**

- [ ] AppsNameExperiment.kt

**How should this be manually tested?**

Make sure the impression event is being sent on the first time you run a version with the experiment (should be sent just once) and that a conversion event is sent every time you click on the apps menu item from any other fragment (pressing it while in the apps fragment to go to the top should not count)

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-512](https://aptoide.atlassian.net/browse/MOB-512)

**Code Review Checklist**
